lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+                                        return Wrap(
+                                          children: [
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.photo_camera,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Take Photos'),
+                                            ),
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.insert_photo,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Browse Album'),
+                                            ),
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.search,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Search Community'),
+                                            ),
+                                          ],
+                                        );
+                                      },
+                                    )
+                                  },
+                                  child: Image.asset(
+                                    widget.tile.imageUrl,
+                                    width: screenSize.width / 2,
+                                  ),
+                                ),
+                              ),
+                            ),
+                            Container(
+                              height: 24,
+                              child: Text(widget.tile.name,
+                                  style: TextStyle(
+                                    fontSize: 12.0,
+                                    fontFamily: "Robotto",
+                                    fontWeight: FontWeight.w500,
+                                  )),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            Card(
+              elevation: 0.0,
+              // margin: const EdgeInsets.fromLTRB(1.0, 8.0, 1.0, 8.0),
+              child: Column(
+                children: <Widget>[
+                  Container(
+                    decoration: BoxDecoration(
+                      border: Border(
+                        bottom: new BorderSide(color: Colors.black12),
+                      ),
+                    ),
```

Comment:
I see here we have a `Column` inside a `Container` inside another `Column` inside a `Card`. Is there a way to simply this deep nesting here to making things a little bit more clearer? Maybe you can abstract some things in a widget to prevent having to repeat code.